pom.xml

```diff
@@ -401,6 +398,9 @@
     </profile>
     <profile>
       <id>spark-2.1</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
       <properties>
         <spark.version>2.1.0</spark.version>
```

Comment:
let us first focus on 2.1.0.  we can support 2.1.1 or 2.2 later as per most of users' requirements.